### ISSUE TITLE
No accessible label on TWINT radio button

### ISSUE THREAD

**Issue by alinekeller** [Twint-AG/twint-woocommerce-extension#57]
Hello,

I just installed the latest version of TWINT Payment for WooCommerce (1.7.5) on a website running with WordPress 7.0 and WooCommerce 10.8.1. I noticed the following accessibility issue:

### Summary
In the payment method list, on the checkout page, the radio button allowing to select TWINT has no accessible label. The `<label>` tag only contains an image with no `alt` attribute. Because of that, screen readers only say "radio button" and users have no idea what the option is. This results in non-compliance with the WCAG for all websites that use the plugin.

The current HTML looks like this:

```
<label class="wc-block-components-radio-control__option" for="radio-control-wc-payment-method-options-twint_regular">
    <input id="radio-control-wc-payment-method-options-twint_regular" class="wc-block-components-radio-control__input" type="radio" name="radio-control-wc-payment-method-options" aria-describedby="radio-control-wc-payment-method-options-twint_regular__content" aria-disabled="false" value="twint_regular">
    <div class="wc-block-components-radio-control__option-layout">
        <div class="wc-block-components-radio-control__label-group">
            <span id="radio-control-wc-payment-method-options-twint_regular__label" class="wc-block-components-radio-control__label">
                <span class="wc-block-components-payment-method-label TWINT">
                    <img src="data:image/svg+xml;base64,…">
                </span>
            </span>
        </div>
    </div>
</label>
```

### Proposed fix
Simply adding the payment method title before the image. It fixes the accessibility issue, and having the word "TWINT" written out, in addition to the logo, makes it clearer for all users.

```
<label class="wc-block-components-radio-control__option" for="radio-control-wc-payment-method-options-twint_regular">
    <input id="radio-control-wc-payment-method-options-twint_regular" class="wc-block-components-radio-control__input" type="radio" name="radio-control-wc-payment-method-options" aria-describedby="radio-control-wc-payment-method-options-twint_regular__content" aria-disabled="false" value="twint_regular">
    <div class="wc-block-components-radio-control__option-layout">
        <div class="wc-block-components-radio-control__label-group">
            <span id="radio-control-wc-payment-method-options-twint_regular__label" class="wc-block-components-radio-control__label">
                <span class="wc-block-components-payment-method-label TWINT">
                    <span class="twint-accessible-label">TWINT</span>
                    <img src="data:image/svg+xml;base64,…">
                </span>
            </span>
        </div>
    </div>
</label>
```

### Additional suggestion
The radio button `<input>` tag has an `aria-describedby` attribute. With other payment methods, this refers to a `<div>` containing the description of the payment method. With the TWINT option, the element with `id="radio-control-wc-payment-method-options-twint_regular__content"` is missing, so the `aria-describedby` attribute refers to nothing. It would be very useful to follow WooCommerce best practices and add the description.

**Comment by nhamtphat** [comment]
Hi @alinekeller, 

Thank you for the detailed report and the proposed fix! We're on it.

Best regards,
Phat

**Comment by nhamtphat** [comment]
Hi @alinekeller

We just released the new version 1.7.6 to address your reported issue. Could you help us verify if it works? Thank you!

**Comment by alinekeller** [comment]
> Hi [@alinekeller](https://github.com/alinekeller)
> 
> We just released the new version 1.7.6 to address your reported issue. Could you help us verify if it works? Thank you!

Hi @nhamtphat 
Thank you for addressing the issue so quickly, it's way better ! Maybe a small visual improvement would be nice, currently there is no margin between the visual label and the TWINT icon. You could just add:

```
width: 100%;
justify-content: space-between;
align-items: center;
```
on `.wc-block-components-payment-method-label.TWINT` (it already has `display: inline-flex`).

Best regards,
Aline

**Comment by nhamtphat** [comment]
Hi @alinekeller , 
Thank you for your suggestion, it will be included in the next release.

Best regards,
Phat

**Comment by alinekeller** [comment]
Hi @nhamtphat ,
Thank you for the visual fix, it looks great !

Best regards,
Aline
